tests/test_check.py

```diff
@@ -13,6 +16,22 @@
 ]
 
 
+@pytest.fixture()
+def malformed_src(tmp_path, src_scratch_path):
+    test_src_path = tmp_path / "src"
+    shutil.copytree(src_scratch_path, test_src_path)
+
+    delete_isolates(test_src_path / OTU_DIR_NAMES[0])
```

Comment:
`src_scratch_path` is already copied to `tmp_path` in the fixture function.

```suggestion
    delete_isolates(src_scratch_path / OTU_DIR_NAMES[0])
```